modules/openapi-generator/src/test/resources/3_0/6150_model_json_inline.yaml

```diff
@@ -0,0 +1,77 @@
+openapi: "3.0.0"
+info:
+  version: 1.0.0
+  title: Test inline response model
+  description: Test inline response model.
+  license:
+    name: MIT
+paths:
+  /foobar:
+    get:
+      operationId: testOperation
+      description: No type property in modelJson of InlineResponse200
+      responses:
+        200:
+          description: InlineResponse200 itself.
+          content:
+            application/json:
+              schema:
+                type: object
+                # OAS3 spec allows to use any value in format field
+                format: unknown
```

Comment:
Same question/comment as what I wrote for line 39 below. What is the purpose of specifying the `format` keyword for an object type in your test?

Reply:
Just to check that `format` property saved in broken schemas. On purpose.